### DISCUSSION TITLE
Workspace 2 gets NET_DESKTOP_NAMES 9

### DISCUSSION THREAD

**Discussion by TheChosenOne1984** [i3/i3#5595] [Q&A]
Hi all,
I don't think this is a i3 issue, but maybe someone can explain this to me.

If I open another workspace using super+2 and then check the output of `xprop -root _NET_DESKTOP_NAMES` the output looks like this:
`_NET_DESKTOP_NAMES(UTF8_STRING) = "1", "9", ""`

If I add more workspaces, for example 3 to 9, the names for the workspaces 3-9 are as expected. But 9 appears twice then.
`_NET_DESKTOP_NAMES(UTF8_STRING) = "1", "9", "3", "4", "5", "6", "7", "8", "9", ""`

Can someone explain what is going on here? Or maybe help me with the right keywords for google? I wasn't lucky so far.

Thank you!

**Comment by orestisfl** [comment]
This is definitely an i3 issue. What's your i3 version, run `i3 --moreversion`.

**Reply by TheChosenOne1984** [reply]
```
Binary i3 version:  4.22 (2023-01-02) © 2009 Michael Stapelberg and contributors
Running i3 version: 4.22 (2023-01-02) (pid 9237)
```

**Reply by orestisfl** [reply]
Great, thanks

Can you open a new bug ticket with full details? Reproduction instructions and log: https://i3wm.org/docs/debugging.html
